src/github-graphql.js

```diff
@@ -148,6 +150,48 @@ class GraphQLClient {
     return this;
   }
 
+  // first type of mutations: add reaction
+  // return boolean value indicating result of action
+  async addReaction({id, content}) {
+    if (DEBUG) {
+      console.log('add reaction for id', id, 'with content', content);
+    }
+    let data, errors;
+    try {
+      ({ data, errors } = await this.client.query(
+        GITHUB_REACTION_ADD_MUTATION,
+        {id, content}
+      ));
+    } catch (error) {
+      return {result: false, msg: error};
+    }
+    if (!data || errors) {
+      return {result: false, msg: errors};
+    }
+    return {result: true, msg: data};
+  }
+
+  // second type of mutations: remove reaction
+  // return boolean value indicating result of action
+  async removeReaction({id, content}) {
+    if (DEBUG) {
+      console.log('remove reaction for id', id, 'with content', content);
+    }
+    let errors;
+    try {
+      ({ errors } = await this.client.query(
+        GITHUB_REACTION_REMOVE_MUTATION,
+        {id, content}
+      ));
+    } catch (error) {
+      return {result: false, msg: error};
+    }
+    if (errors) {
+      return {result: false, msg: errors};
```

Comment:
Did GitHub sent 2xx status even if there's an error? and that causes it to not being `catch`ed?

What kind of error fall into this?

Reply:
When you try to remove a reaction from a comment you've never reacted before, GitHub will return a failure message (not 2xx).

Reply:
But with 2xx error code, as in success?